source/outlines/2016-04-27.markdown

```diff
@@ -0,0 +1,66 @@
+---
+title: 20160427
+layout: outline
+---
+
+## All
+
+### Word of the Day
+* afterclap - noun - an unexpected, often unpleasant sequel to a matter that
+had been considered closed.
+
+
+## 1510
+
+### Job Talk (9:00 - 10:30)
+
+Classroom C.
+
+### One on Ones (10:30 - 12:00)
```

Comment:
I'd like to also make this The Plan evals - they actually end up being One on Ones